generate-test-vector.py

```diff
@@ -509,167 +403,30 @@ def generate_multiple_outputs_with_labels_tests():
-        sender['expected']['outputs'] = outputs
-        recipient['given']['outputs'] = outputs
-
-        add_to_wallet = reference.scanning(
-            scan1,
-            Spend1,
-            A_sum,
-            deterministic_nonce,
-            [ECPubKey().set(bytes.fromhex(pub)) for pub in outputs],
-            labels={(reference.generate_label(scan1, l)*G).get_bytes(False).hex():reference.generate_label(scan1, l).hex() for l in labels[i]},
-        )
-        for o in add_to_wallet:
-
-            pubkey = ECPubKey().set(bytes.fromhex(o['pub_key']))
-            full_private_key = spend1.add(
-                bytes.fromhex(o['priv_key_tweak'])
-            )
-            if full_private_key.get_pubkey().get_y()%2 != 0:
-                full_private_key.negate()
-
-            sig = full_private_key.sign_schnorr(msg, aux)
-            assert pubkey.verify_schnorr(sig, msg)
-            o['signature'] = sig.hex()
-
-        recipient['expected']['outputs'] = add_to_wallet
-        test_case['sending'].extend([sender])
-        test_case['receiving'].extend([recipient])
-        test_case["comment"] = comments[i]
-        test_cases.append(test_case)
-
-    return test_cases
-
-
-def generate_more_labels_tests():
-
-    msg = hashlib.sha256(b'message').digest()
-    aux = hashlib.sha256(b'random auxiliary data').digest()
-    G = ECKey().set(1).get_pubkey()
-    recipient_test_cases = []
-    outpoints = [
-            ("f4184fc596403b9d638783cf57adfe4c75c605f6356fbc91338530e9831e9e16", 0),
-            ("a1075db55d416d3ca199f55b6084e2115b9345e16c5cf302fc80e9d5fbf5d48d", 0),
-    ]
-    sender_bip32_seed = 'deadbeef'
-    i1, I1 = get_key_pair(0, seed=bytes.fromhex(sender_bip32_seed))
-    i2, I2 = get_key_pair(1, seed=bytes.fromhex(sender_bip32_seed))
-    input_priv_keys = [(i1, False), (i2, False)]
-    input_pub_keys = [I1, I2]
-
-    recipient_bip32_seed = 'f00dbabe'
-    scan1, spend1, Scan1, Spend1 = derive_silent_payment_key_pair(bytes.fromhex(recipient_bip32_seed))
-    address = reference.encode_silent_payment_address(Scan1, Spend1, hrp=HRP)
-    scan2, spend2, Scan2, Spend2 = derive_silent_payment_key_pair(bytes.fromhex(recipient_bip32_seed) + b'01')
-    address2 = reference.encode_silent_payment_address(Scan1, Spend1, hrp=HRP)
-    l1 = 1
-    l2 = 1337
-    labels = [[l1, l2]]
-    label_address_one = reference.create_labeled_silent_payment_address(scan1, Spend1, m=l1, hrp=HRP)
-    label_address_two = reference.create_labeled_silent_payment_address(scan1, Spend1, m=l2, hrp=HRP)
-    label_address_one2 = reference.create_labeled_silent_payment_address(scan2, Spend2, m=l1, hrp=HRP)
-    label_address_two2 = reference.create_labeled_silent_payment_address(scan2, Spend2, m=l2, hrp=HRP)
-    addresses1 = [label_address_one, address, address2, label_address_one, label_address_one2, label_address_one, label_address_two2, label_address_two, label_address_two, label_address_two2]
```

Comment:
in 404fe33e9318f200959e76875505edec7be10e90

Is it on purpose you dropped this test?

Reply:
Will need to look again, but IIRC, this test wasn't adding any coverage by having 2 recipients, so was complicated in that it needed to two recipients but for no real benefit. Will take a look again to confirm, tho!